ui/admin/app/routes/application.js

```diff
@@ -20,6 +20,7 @@ export default class ApplicationRoute extends Route {
   @service confirm;
   @service router;
   @service intl;
+  @service features;
```

Comment:
None blocking comment: where `features` service lives? I see `feature-editions` within `app/services/feature-edition` but not `features`.

Reply:
https://www.npmjs.com/package/ember-feature-flags